src/components/Garden/Stake/StakingMovements.js

```diff
@@ -16,45 +16,38 @@ import {
 } from '@1hive/1hive-ui'
 import { useGardenState } from '@providers/GardenState'
 import {
-  STAKING_SCHEDULED,
-  STAKING_CHALLENGED,
-  STAKING_COMPLETED,
-  STAKING_CANCELLED,
-  STAKING_SETTLED,
-  COLLATERAL_LOCKED,
-  COLLATERAL_CHALLENGED,
-  COLLATERAL_AVAILABLE,
-  COLLATERAL_SLASHED,
-  STAKING_STATUSES,
-  COLLATERAL_STATUSES,
+  StakingType,
+  StakingCollateralType,
+  StakingStatusesMap,
+  CollateralStatusesMap,
```

Comment:
the same comment as [here](https://github.com/1Hive/gardens-ui/pull/759/files?authenticity_token=t1pgHOdlLaAxMP1ru8q1pA1scgbSqcddQF8xzbmiOLv4%2Fcc3yBArx57VTsi1Nuw6lI37uRo8cTwtVKEJeHjnwg%3D%3D&file-filters%5B%5D=.js&file-filters%5B%5D=.lock&file-filters%5B%5D=.ts&file-filters%5B%5D=.tsx#r776804492)

Reply:
Done ✅